lane.py

```diff
@@ -72,7 +72,7 @@
     Library,
     LicensePool,
     LicensePoolDeliveryMechanism,
-    MaterializedWorkWithGenre as work_model,
+    MaterializedWorkWithGenre as mw,
```

Comment:
Since `MaterializedWorkWithGenre` is no longer the only "work model" in play in this file, I changed the import name. This causes most of the line diffs in this branch.